bundler/lib/bundler/templates/newgem/newgem.gemspec.tt

```diff
@@ -43,6 +43,9 @@ Gem::Specification.new do |spec|
 
   # Uncomment to register a new dependency of your gem
   # spec.add_dependency "example-gem", "~> 1.0"
+<%- if config[:zeitwerk] -%>
+  spec.add_dependency "zeitwerk"
+<%- end -%>
```

Comment:
Minor comment, feel free to ignore: maybe this could be moved below the rust condition, so if both are enabled, the zeitwerk `add_dependency` would come last alphabetically.